package.json

```diff
@@ -17,15 +22,15 @@
     "format:prettier": "prettier \"**/*.{cjs,html,js,json,md,ts}\" --write",
     "ignore-sync": "ignore-sync .",
     "lint": "eslint \"**/*.{js,ts}\"",
-    "nuke": "rm -rf node_modules && npm install && lerna exec 'rm -rf node_modules' && lerna bootstrap && npm run clean",
-    "upgrade": "rm -rf node_modules package-lock.json && npm install && lerna exec 'rm -rf node_modules package-lock.json' && lerna bootstrap && npm run clean",
+    "nuke": "rm -rf node_modules && npm exec -ws -c 'rm -rf node_modules' && npm ci",
```

Comment:
This is difference from the previous nuke, which ran a regular `npm i` via `lerna bootstrap`. I also think it should be basically unnecessary as `npm ci` should put all node_modules in the state described by the package-lock. The only case that's different is if you modified files in node_modules/

Reply:
Deleted the script